File: FactuMex/ClientApp/src/app/usuario/usuario.ts
export class Usuario {
    constructor(
        public IDUsuario: string,
        public Nombre: string,
        public Email: string,
        public Pwd: string,
        public Cedula: string,
        public Telefono: string,
        public Foto: string,
        public Borrado: string,
        public usr: string,
        public token: string
    ) {
    }
}
export class Notificacion {
    constructor(
        public Msj: string,
        public Src: string
    )
    {
    }
}
export class itemPermiso {
    constructor(
        public IDPermiso: string,
        public Permiso: string,
        public Grupo: string
    ) {
    }
}
export class itemPlantilla {
    constructor(
        public IDPermisoxUsuario: string,
        public IDUsuario: string,
        public Permiso: string) { }
}

export class DTOPermisos {
    constructor(
        public Usuario: itemPlantilla[],
        public Plantilla: itemPermiso[]) { }
}

File: FactuMex/Models/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CallCenter.Models
{
    public class Usuario
    {
        public string IDUsuario { get; set; }
        public string Nombre { get; set; }
        public string Email { get; set; }
        public string Pwd { get; set; }
        public string Cedula { get; set; }
        public string Telefono { get; set; }
        public string Foto { get; set; }
        public string Borrado { get; set; }
    }
}
